Traceback (most recent call last):
  File "main.py", line 370, in <module>
    main()
  File "main.py", line 246, in main
    dict_ensemble = evaluate_ga('/csv/' + name[:-4] + '.csv', labels, train_data, test_data, sys.argv[3])
  File "/mnt/campus/math/cluster/research/slurm-pr1/work_root/mmalekis/MVVGE/src/ga/evaluate_ga.py", line 433, in evaluate_ga
    , train_x_lr_st_reduction, train_x_lr_reduction, train_y_lr_reduction, test_x_lr_st_reduction, test_x_lr_reduction, test_y_lr_reduction, reduction_variance, reduction_variance_ratio = process.mds(
  File "/mnt/campus/math/cluster/research/slurm-pr1/work_root/mmalekis/MVVGE/src/data_handeling/Preprocess.py", line 403, in mds
    mds_train.fit(train)
  File "/work/mmalekis/.conda/envs/gennn/lib/python3.8/site-packages/sklearn/manifold/_mds.py", line 488, in fit
    self.fit_transform(X, init=init)
  File "/work/mmalekis/.conda/envs/gennn/lib/python3.8/site-packages/sklearn/manifold/_mds.py", line 534, in fit_transform
    self.embedding_, self.stress_, self.n_iter_ = smacof(
  File "/work/mmalekis/.conda/envs/gennn/lib/python3.8/site-packages/sklearn/manifold/_mds.py", line 285, in smacof
    results = Parallel(n_jobs=n_jobs, verbose=max(verbose - 1, 0))(
  File "/work/mmalekis/.conda/envs/gennn/lib/python3.8/site-packages/joblib/parallel.py", line 1056, in __call__
    self.retrieve()
  File "/work/mmalekis/.conda/envs/gennn/lib/python3.8/site-packages/joblib/parallel.py", line 935, in retrieve
    self._output.extend(job.get(timeout=self.timeout))
  File "/work/mmalekis/.conda/envs/gennn/lib/python3.8/site-packages/joblib/_parallel_backends.py", line 542, in wrap_future_result
    return future.result(timeout=timeout)
  File "/work/mmalekis/.conda/envs/gennn/lib/python3.8/concurrent/futures/_base.py", line 439, in result
    return self.__get_result()
  File "/work/mmalekis/.conda/envs/gennn/lib/python3.8/concurrent/futures/_base.py", line 388, in __get_result
    raise self._exception
joblib.externals.loky.process_executor.TerminatedWorkerError: A worker process managed by the executor was unexpectedly terminated. This could be caused by a segmentation fault while calling the function or by an excessive memory usage causing the Operating System to kill the worker.

The exit codes of the workers are {SIGKILL(-9)}
slurmstepd: error: Detected 1 oom-kill event(s) in StepId=4648.batch. Some of your processes may have been killed by the cgroup out-of-memory handler.
